docs/pages/faq.tsx

```diff
@@ -0,0 +1,105 @@
+import clsx from 'clsx';
+import matter from 'gray-matter';
+import Link from 'next/link';
+import Task from 'taskarian';
+import { getFilesFromPath, getMarkDownWithMeta } from '../lib';
+import { Page, requireFrontmatterDuringBuild, unsafeMarkdownFromContent } from '../Types/guide';
+import { taskToStaticProps, WithNavTree, withNavTreeStaticProp } from '../Types/NavTree';
+import { pipe } from '@kofno/piper';
+import { GetStaticProps } from 'next';
+import PageTitle from '../components/PageTitle';
+
+interface Props {
+  pages: Page[];
+}
+
+const FAQ: React.FC<Props> = ({ pages }) => {
+  return (
+    <>
+      <PageTitle title="FAQs" />
+      <div className={clsx('not-prose relative lg:block lg:flex-none')}>
+        <div
+          className={clsx(
+            'sticky top-[4.5rem] -ml-0.5 h-[calc(100vh-4.5rem)]',
+            'overflow-y-auto py-16 pl-0.5'
+          )}
+        >
+          <nav
+            className={clsx('w-64 pr-8 text-base lg:text-sm xl:w-72 xl:pr-16')}
+            aria-labelledby="secondary-nav-header"
+          >
+            <ul className={clsx('space-y-9')}>
+              <li>
+                <h1
+                  className={clsx('font-display font-medium text-slate-900 dark:text-white')}
+                  id="secondary-nav-header"
+                >
+                  Frequently Asked Questions
+                </h1>
+                <ul
+                  className={clsx(
+                    'not-prose lg:border-slate-20',
+                    'mt-6 space-y-2 border-l-2',
+                    'border-slate-100 dark:border-slate-800',
+                    'lg:mt-4 lg:space-y-4'
+                  )}
+                >
+                  {pages.map(({ slug, frontmatter: { title } }) => (
+                    <li key={slug} className={clsx('relative')}>
+                      <Link href={`/faq/${slug}`}>
+                        <a
+                          href="#"
```

Comment:
```suggestion
```

Let's remove this. If the anchor tag doesn't have an `href` the `Link` component will fill it in in the html, so the user doesn't need JS to view the page.